tests/test.rs

```diff
@@ -75,17 +75,6 @@ fn successful_run_has_no_backtrace() {
     insta::assert_snapshot!(run_result.output);
 }
 
-#[test]
-// this test should not be run by default, as it requires the target hardware to be present
-#[ignore]
-fn successful_run_can_enforce_backtrace() {
```

Comment:
why is this deleted and not adapted?

Reply:
Didn't realize at first that we keep the functionality (under a different name). Adapted!